### ISSUE TITLE
Add outcome related to Mood

### ISSUE THREAD

**Issue by SuzanneKTaylor** [w3c/wcag3#88]
The [Introduction to the May 2024 draft of WCAG 3.0](https://www.w3.org/blog/2024/the-wcag-3-working-draft-update-is-ready-for-your-review/) asked  "What outcomes needed to make web content accessible are missing?"

While Disturbing Content and Triggers are included, often content can create a mood without necessarily having acute content items that can be identified as "disturbing" or as a "trigger," so an outcome like this might be beneficial to help users navigate the sea of content that exists today and that may grow exponentially through AI:

Mood Impact
Information about the mood content is likely to encourage is available or can be changed/selected by the user.

**Comment by brennanyoung** [comment]
are you thinking of stuff like "Sober" / "Professional" / "Light-hearted" / "Romantic" / "Sinister" ? Or something more long-form?

Looking at the current set of outcomes, I still lack one which pertains to **connotative** aspects of content (as distinct from denotative, or "purely decorative" aspects), and I think your suggestion might fit the bill very nicely.

Connotation is not quite dentical to mood, but there is significant overlap. Would some kind of synthesis of these two ideas be viable? Mood is perhaps a more widely-understood word.

A lot of content (especially decorative content) has a distinct mood/connotation, but a detailed description of that content can add unwelcome noise, and may be liable to "kill" the mood. (E.g. the "erotic" or the "sinister" is often about what goes **unmentioned**), so in many cases it would be useful to be able to flag a mood/connotation **in lieu of an actual description**.

Not sure about offering change/selection in this area, but it could certainly be that I want the tutorial without the jokes. I think this is a bit different, though.

**Comment by SuzanneKTaylor** [comment]
I was thinking about longer-form content and mental health. So, for example, if it's an augment reality experience that is considered comedy, but if the user is working to be less anxious, they might benefit from knowing upfront if the experience generally generates anxiety.

I appreciate what you are saying as well. "Grinning clown with round red nose" doesn't really give the connotative info - is this a funny thing for young children or is it a horror story - we wouldn't know. You can use connotative language to parallel the connotative image for blind users. That might be one technique. However, for users with cognitive differences, connotative information could be helpful, and those users might not read the description. In addition, for blind users reading in a second or third language, the connotative language nuances could be more likely to be missed compared with seeing the connotative meaning in an image. So a way to highlight items (images, text, slogans) with particular connotations could be beneficial. Specific tagging for this, might also help where the author is struggling to get their connotations across. (e.g. "I was joking. I'm not really angry")

**Comment by detlevhfischer** [comment]
> Mood Impact
Information about the mood content is likely to encourage is available or can be changed/selected by the user.

> So a way to highlight items (images, text, slogans) with particular connotations could be beneficial.

@SuzanneKTaylor Are you suggesting adding a meta-level of information to just about anything? Where to stop? As you comment youself regarding the clown example, "mood" is not simply a property of content but highly dependent on the user and certainly also on the content within other elements. As such annotation needs to be exposed visually and programmatically, it also adds more stuff to the "sea of content" that may be helpful for some but would certainly be distracting and unhelpful for others. I do not see how this could become a general measurable outcome.

**Comment by SuzanneKTaylor** [comment]
"So a way to highlight items (images, text, slogans) with particular connotations could be beneficial." was acknowledging the separate idea that Brennan brought up - relating to things like alt-text.

"Information about the mood content is likely to encourage is available or can be changed/selected by the user." would be metadata for longer form content. This Github issue was to suggest metadata for longer form content. (a movie, a television show, an ebook, a VR experience) Something that users spend quite a bit time within. It would be about the mood/effect on mental health the content creates in users (or would be expected to create in users). 

There are many complexities to consider if the idea is generally accepted by the group as interesting to explore. As experiences become easier and easier to generate and manipulate, the need to know (or set) the mood in advance becomes more necessary.

**Comment by detlevhfischer** [comment]
> "So a way to highlight items (images, text, slogans) with particular connotations could be beneficial." was acknowledging the separate idea that Brennan brought up - relating to things like alt-text.

Ok, perhaps I misunderstood. This would then not add noise since by default it would be a user agent- or OS-supplied setting to expose connotative annotations of mood if the user has activated such a setting? As with all settings-supplied things, a large part of users is then likely to not realise that there is such a setting, and miss out on mood, but who knows - it may become as popular as dark mode.

As metadata for longer audiovisual content, providing information on mood may have benefits. I guess there is a gray area between mood and content considered a trigger for some.

**Comment by patrickhlauke** [comment]
Not going to lie, I slightly dread a situation in the future where to evaluate whether or not content is deemed accessible under WCAG 3 (to whatever conformance level/score/etc), auditors will need to check for this sort of meta information as well

**Comment by SuzanneKTaylor** [comment]
I think either of these two outcomes might be possible through AT. In that case, perhaps WCAG would only be concerned with whether that future AT had the access it needs to pre-screen. For example, for the long form content (let's say a movie), can the AT go through the whole movie and judge it, or does it get stopped at the first ad.

**Comment by SuzanneKTaylor** [comment]
> This would then not add noise since by default it would be a user agent- or OS-supplied setting to expose connotative annotations of mood if the user has activated such a setting?

Yes, it could be revealed through a browser settings visually, and through screen reader settings for this additional enhancement to alt-text. So the screen reader output might be "sinister mood graphic, Grinning clown with round red nose" - could be done today with role-description, but then you couldn't opt-out.

**Comment by ericwbailey** [comment]
This would be impossible to quantify, and that's before you even factor in cultural considerations.

**Comment by SuzanneKTaylor** [comment]
Well Brennan's idea relates to the intended mood or connotation, so that could be determined by "the author".

For my suggestion, you are right. One person might find a story of extreme hardship depressing, while another might find it to be a call to action, and still another might like how their circumstance looks in comparison. This would likely need to be calibrated by the user within an AT.

However, I personally think new AT is needed for many of the outcomes.
